openresty-build-tools/kong-ngx-build

```diff
@@ -61,6 +61,10 @@ main() {
         OPENSSL_SHA=$2
         shift 2
         ;;
+      --with-openssl-fips)
+        OPENSSL_FIPS_VER=2.0.16
```

Comment:
I suspect we want this to eventually be pinned in `kong/.requirements` which would look something like change this line to `OPENSSL_FIPS_VER=$2`

and pull the variable in the Makefile and send it through to kong-ngx-build with edits:
https://github.com/Kong/kong-build-tools/blob/4.3.6/Makefile#L37
https://github.com/Kong/kong-build-tools/blob/4.3.6/dockerfiles/Dockerfile.openresty#L68
https://github.com/Kong/kong-build-tools/blob/4.3.6/build-openresty.sh#L47

Reply:
The commit and PR explain why this is hard-coded.